File: benchmarks/bench_1.go
package main

import (
	"fmt"
	"time"
)

func loop_10m() int64 {
    x := int64(0)
    for x < 10000000 {
        x += 1
    }
    return x
}

func main() {
    t1 := time.Now()
    res := loop_10m()
    t2 := time.Now()
    fmt.Println(res)
    fmt.Printf("%.6f\n", t2.Sub(t1).Seconds())
}

File: benchmarks/bench_1.py
import time
import sys
sys.setrecursionlimit(200000)

def loop_10m():
    x = 0
    while x < 10000000:
        x += 1
    return x


t1 = time.perf_counter()
res = loop_10m()
t2 = time.perf_counter()
print(res)
print(f"{t2 - t1:.6f}")
